src/Lesson_9/TaskTwo/LoginForm.java

```diff
@@ -0,0 +1,52 @@
+package Lesson_9.TaskTwo;
+
+import java.util.Scanner;
+
+public class LoginForm {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner((System.in));
+
+        System.out.println("Enter login: ");
+        String login = scanner.nextLine();
+
+        System.out.println("Enter password: ");
+        String password = scanner.nextLine();
+
+        System.out.println("Confirm password: ");
+        String confirmPassword = scanner.nextLine();
+
+        if(login(login, password, confirmPassword)){
+            System.out.println("You are successfully logged.");
+        } else {
+            System.out.println("You haven't logged.");
+        }
+    }
+
+    public static boolean login(String login, String password, String confirmPassword) {
+
+        try {
+            WrongLoginException exceptionOne = new WrongLoginException("The login cannot contain more than 20 characters. Please try again.");
```

Comment:
We don't need to create objects that may not be in use. Throw exceptions only when critical situation began